<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>segmenti</class>
 <widget class="QWidget" name="segmenti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoAI - Segmentar Imagen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="paginador">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>OPCIONES DE CONFIGURACION</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>168</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="defecto">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777209</height>
            </size>
           </property>
           <property name="text">
            <string>Parametros por defecto</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="2" colspan="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>168</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>1.- Puntos por iteracion</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QSpinBox" name="Points_per_batch">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>64</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>     Points_per_batch</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>2.- Densidad de muestreo. Puntos por lado</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QSpinBox" name="Points_per_side">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>     Points_per_side</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>3.- Umbral de filtrado</string>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QDoubleSpinBox" name="Pred_iou_thresh">
           <property name="maximum">
            <double>0.990000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.880000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>     Pred_iou_thresh</string>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>4.- Umbral de estabilidad</string>
           </property>
          </widget>
         </item>
         <item row="10" column="4">
          <widget class="QDoubleSpinBox" name="Stability_score_thresh">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.950000000000000</double>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>         Stability_score_thresh</string>
           </property>
          </widget>
         </item>
         <item row="11" column="3">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="12" column="0" colspan="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>5.- Recorte capas. &gt;0 mayor procesamiento</string>
           </property>
          </widget>
         </item>
         <item row="12" column="4">
          <widget class="QSpinBox" name="Crop_n_layers">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>         Crop_n_layers</string>
           </property>
          </widget>
         </item>
         <item row="13" column="3">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>6.- Factor de reduccion de escala</string>
           </property>
          </widget>
         </item>
         <item row="14" column="4">
          <widget class="QSpinBox" name="Crop_n_points_downscale_factor">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="15" column="0" colspan="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>         Crop_n_points_downscale_factor</string>
           </property>
          </widget>
         </item>
         <item row="15" column="3">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2" rowspan="2" colspan="3">
          <widget class="QComboBox" name="listVector">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="opVector">
           <property name="text">
            <string>  0.- Utilizar capa de puntos (EXPERIMENTAL)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="icon_portada_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>220</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>220</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="opBasicas">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>   INICIO</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="10" column="5">
        <widget class="QPushButton" name="cancelar">
         <property name="text">
          <string>CANCELAR</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Banda Rojo (Red)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="bandR">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Banda Verde (Green)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="bandG">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Banda Azul (Blue)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="bandB">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="3" colspan="3">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>215</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="tinyhqop">
         <property name="text">
          <string>Utilizar modelo HQ ligero </string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="brmodelo">
         <property name="text">
          <string>Ubique el archivo</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="ruta_file">
         <property name="text">
          <string>................................</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="listImagen">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Ruta al archivo del modelo</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4" rowspan="6" colspan="2">
        <widget class="QLabel" name="icon_portada">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>220</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>220</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Nombre del Modelo</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Seleccionar la imagen</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>MODELO SAM-------------------------------</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Procesamiento</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="device">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>CPU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GPU si es posible</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="nmodelo">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>vit_h</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vit_l</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vit_b</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="4">
        <widget class="QPushButton" name="ejecutar">
         <property name="text">
          <string>PROCESAR</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CONFIGURAR EL MODELO</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="3">
        <widget class="QPushButton" name="opAvanzadas">
         <property name="text">
          <string>       OPCIONES AVANZADAS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
